Tests/SecureEnclaveTests.swift

```diff
@@ -40,7 +40,18 @@ class SecureEnclaveTests: XCTestCase
         
         valet.removeObject(forKey: key)
     }
-    
+
+    // MARK: Initialization
+
+    func test_init_createsCorrectBackingService() {
+        let identifier = ValetTests.identifier
+
+        SecureEnclaveAccessControl.allValues().forEach { accessControl in
+            let backingService = SecureEnclaveValet.valet(with: identifier, accessControl: accessControl).service
+            XCTAssertEqual(backingService, Service.standard(identifier, .secureEnclave(accessControl)))
+        }
+    }
```

Comment:
Shouldn't we also include shared access tests here (and in `SinglePromptSecureEnclaveTests`)?